CHANGELOG.md

```diff
@@ -2,9 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
-## 0.2.0 - unreleased
+## [Unreleased]
 
-## 0.1.0 - 2021-09-01
+### Added
+- Pod placement scheduler with role+group anti-affinity and K8S stored history.
```

Comment:
Can you check e.g. [agent](https://github.com/stackabletech/agent/blob/main/CHANGELOG.md) changelog to be consistent? Basically linking the PR the change / add / delete occurs?